account_payment_order/models/payment_order.py

```diff
@@ -0,0 +1,172 @@
+                            states={'done': [('readonly', True)]},
+                            default=lambda self:
+                            self.env['ir.sequence'].
+                            next_by_code('payment.order'),
+                            copy=False)
+
+    mode = fields.Many2one('payment.mode', 'Payment Mode', select=True,
+                           required=1, states={'done': [('readonly', True)]},
+                           help='Select the Payment Mode to be applied.')
+
+    state = fields.Selection([('draft', 'Draft'),
+                              ('cancel', 'Cancelled'),
+                              ('open', 'Confirmed'),
+                              ('done', 'Done')], 'Status', select=True,
+                             copy=False, default='draft',
+                             help='When an order is placed the status is '
+                             '\'Draft\'.\n Once the bank is confirmed the '
+                             'status is set to \'Confirmed\'.\n'
+                             'Then the order is paid the status is \'Done\'.')
+
+    line_ids = fields.One2many('payment.line', 'order_id',
+                               'Payment lines',
+                               states={'done': [('readonly', True)]})
+
+    total = fields.Float(compute='_compute_total', string="Total", store=True)
+
+    user_id = fields.Many2one('res.users', 'Responsible', required=True,
+                              states={'done': [('readonly', True)]},
+                              default=lambda self: self.env.uid)
+
+    date_prefered = fields.\
+        Selection([('now', 'Directly'), ('due', 'Due date'),
+                   ('fixed', 'Fixed date')
+                   ], "Preferred Date", change_default=True, default='due',
+                  required=True, states={'done': [('readonly', True)]},
+                  help="Choose an option for the Payment Order:'Fixed'"
+                       "stands for a date specified by you.'Directly' stands "
+                       "for the direct execution. 'Due date' stands for the "
+                       "scheduled date of execution.")
+
+    date_done = fields.Date('Execution Date', readonly=True)
+
+    company_id = fields.Many2one('res.company', related='mode.company_id',
+                                 string='Company', store=True, readonly=True)
+
+    entries_test = fields.Many2many('account.move.line', 'test_line_pay_rel',
+                                    'pay_id', 'line_id')
+
+    payment_order_type = fields.Selection(
+        [('payment', 'Payment'), ('debit', 'Direct debit')],
+        'Payment order type', required=True, default='payment',
+        readonly=True, states={'draft': [('readonly', False)]})
+
+    mode_type = fields.Many2one('payment.mode.type', related='mode.type',
+                                string='Payment Type')
+
+    @api.one
+    def set_to_draft(self):
+        self.write({'state': 'draft'})
+#         self.create_workflow()
```

Comment:
I see that you have removed workflow. Wise decision! It's a nonsense here. But please remove all commented lines from the final result. When you have migrated all the commit history, we were been able to see the removal in the diff